src/MCPClient/lib/clientScripts/extract_contents.py

```diff
@@ -145,39 +191,52 @@ def main(job, transfer_uuid, sip_directory, date, task_uuid, delete=False):
 
         if not exitstatus == 0:
             # Dang, looks like the extraction failed
-            job.pyprint('Command', command.description, 'failed!', file=sys.stderr)
+            job.pyprint('Command',
+                        command.description,
+                        'failed!',
+                        file=sys.stderr)
```

Comment:
Perhaps this is better? We don't have to go all vertical, just enough so we don't exceed 79 columns.

```python
            job.pyprint('Command', command.description, 'failed!',
                        file=sys.stderr)
```